<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionDialog</class>
 <widget class="QDialog" name="VersionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>58</y>
     <width>105</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/picture/setting_interface/Rectangle 252.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="icon">
    <iconset resource="picture.qrc">
     <normaloff>:/new/picture/setting_interface/return.png</normaloff>:/new/picture/setting_interface/return.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>21</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_version">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>114</y>
     <width>435</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Wareversion">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>170</y>
     <width>435</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>10</y>
     <width>64</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="btn_label1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>121</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="btn_label2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>177</y>
     <width>95</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="picture.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ReturnButton</sender>
   <signal>clicked()</signal>
   <receiver>VersionDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
